test/testbot.js

```diff
@@ -22,8 +22,10 @@ module.exports = function createSbot(opts = {}) {
   const keys = opts.keys || ssbKeys.loadOrCreateSync(path.join(dir, 'secret'))
 
   const stack = SecretStack({ appKey: caps.shs })
-    .use(require('ssb-db2'))
+    .use(require('ssb-db2/core'))
```

Comment:
the README says that you can use `ssb-db2` >= 5, the dependences say > 3...
do all of these have the latest `ssb-box2` in them by default?

I know they don't because I'm about to open a PR updating the 6...

Reply:
Hmm, yeah feel free to update these. I don't think there's a point in supporting older ssb-db2 versions, because ssb-meta-feeds is so new.